root/etc/s6-overlay/s6-rc.d/init-config-immich/run

```diff
@@ -7,11 +7,14 @@ mkdir -p \
     /import \
     /run/immich
 
+# check if /photos is empty
```

Comment:
why bother if the folder is empty?

Reply:
Otherwise, the CI fails

Reply:
> Otherwise, the CI fails

with what error?

Reply:
je n'arrive pas à retrouver l'erreur :thinking:

Reply:
really make me whip out the translator

Reply:
Lol, "can't find the error"

Reply:
Here you go `Error: EACCES: permission denied, mkdir '/photos/library' `

Reply:
thats /photos not /import?

Reply:
`find /photos -maxdepth 1 -type d -exec chown abc:abc {} \;`

Reply:
Yep, the issue is `/photos`. Someone reported a startup time which took 1h (he has 400 GB of assets)